src/app/admin/pages/learning-objects/learning-objects.component.html

```diff
@@ -1,11 +1,14 @@
 <clark-admin-content-wrapper>
   <div page-title>Learning Objects</div>
-  <clark-admin-search-bar search></clark-admin-search-bar>
+  <clark-admin-search-bar search 
+    [placeholder]="searchBarPlaceholder"
+    (userInput)="getLearningObjects($event)"
+  ></clark-admin-search-bar>
   <div filter-options>filter options yo</div>
 
   <div class="list" main>
```

Comment:
This is a nitpick, but I think it might be helpful to take the whole list div and move it into its own component. It would bring this components template size down a bit, and create a better separation of concerns by leaving one component to compose the page and another to only take charge of displaying the list. I'm interested in other's thoughts on this as well.